meta-rokers/recipes-rokers/images/rokers-image-base.bb

```diff
@@ -1,5 +1,8 @@
 # rokers-image-base
 
+DISTRO_NAME = "Yocto Rokers Distro"
```

Comment:
I am not quite sure the variables defined here will also cause `/etc/issue` to be updated accordingly.

See https://github.com/robotrokers/rokers-yocto-distro/issues/10#issuecomment-313423839